cmd/pbstatuser/main.go

```diff
@@ -135,10 +141,13 @@ func checkSourceAvailabilityAWS(ctx context.Context) {
 			sr.Status = kafka.StatusAvaliable
 			chSend <- sr
 		}
+		metrics.IncTotalAvailablilityCheckReqs(models.ProviderTypeAWS, "statuser", sr.Status, err)
+		metrics.ObserveAvailablilityCheckReqsDuration(models.ProviderTypeAWS, start, err)
 	}
 }
```

Comment:
Here the same, start must be stored when a new variable is received from the channel.

Reply:
Yes, if I am correct.